pkg/quarterdeck/config/config.go

```diff
@@ -17,16 +21,18 @@ import (
 // Quarterdeck API service. This is the top-level config, all sub configurations need
 // to be defined as properties of this Config.
 type Config struct {
-	Maintenance  bool                `default:"false"`                                    // $QUARTERDECK_MAINTENANCE
-	BindAddr     string              `split_words:"true" default:":8088"`                 // $QUARTERDECK_BIND_ADDR
-	Mode         string              `default:"release"`                                  // $QUARTERDECK_MODE
-	LogLevel     logger.LevelDecoder `split_words:"true" default:"info"`                  // $QUARTERDECK_LOG_LEVEL
-	ConsoleLog   bool                `split_words:"true" default:"false"`                 // $QUARTERDECK_CONSOLE_LOG
-	AllowOrigins []string            `split_words:"true" default:"http://localhost:3000"` // $QUARTERDECK_ALLOW_ORIGINS
-	Database     DatabaseConfig
-	Token        TokenConfig
-	Sentry       sentry.Config
-	processed    bool // set when the config is properly processed from the environment
+	Maintenance   bool                `default:"false"`                                                 // $QUARTERDECK_MAINTENANCE
+	BindAddr      string              `split_words:"true" default:":8088"`                              // $QUARTERDECK_BIND_ADDR
+	Mode          string              `default:"release"`                                               // $QUARTERDECK_MODE
+	LogLevel      logger.LevelDecoder `split_words:"true" default:"info"`                               // $QUARTERDECK_LOG_LEVEL
+	ConsoleLog    bool                `split_words:"true" default:"false"`                              // $QUARTERDECK_CONSOLE_LOG
+	AllowOrigins  []string            `split_words:"true" default:"http://localhost:3000"`              // $QUARTERDECK_ALLOW_ORIGINS
+	VerifyBaseURL string              `split_words:"true" default:"https://auth.rotational.app/verify"` // $QUARTERDECK_VERIFY_BASE_URL
```

Comment:
We don't have to do this now, but if we have more configured context, we'll probably want to put this in its own config struct.